### PR TITLE
fix(nsfw): don't allow nsfw images to be sent

### PR DESCRIPTION
**Pull request by Jekrimo**
**What this PR does** 📖
Blocks sending images deemed nsfw and shows an alert notifying user that file/s were not sent that were nsfw

**Which issue(s) this PR fixes** 🔨
AP-285

**Special notes for reviewers** 🗒️
Alert should pop up on it's own and auto close after 5ish seconds, the user can also click the close on the alert to close before then.

### CODE REVIEW

**Comment by phillsatellite** [comment]
Tested and never had a alert pop up, the image would be blured when uploading and send button was disabled 

testnsfw

**Comment by Jekrimo** [comment]
> Tested and never had a alert pop up, the image would be blured when uploading and send button was disabled
> 
> testnsfw

Fixed! I'm a big dummy and was always uploading multiple files to run the check and totally didn't account for a singular nsfw file lol

**Review by phillsatellite — APPROVED** [review]
Boom alert message is showing up now!

File: components/views/files/upload/Upload.html [hunk 1]
```diff
@@ -10,6 +11,17 @@
       <TypographyError :text="$t('errors.chat.drop_file_count')" />
       <x-icon size="1x" class="has-text-danger" @click="cancelUpload" />
     </div>
+    <div v-if="alertNsfw" class="file-error">
+      <TypographyError :text="$t('errors.chat.contains_nsfw')" />
```
Comment (stavares843) [inline components/views/files/upload/Upload.html:15]: Captura de ecrã 2021-12-30, às 16 34 07

we need to add the string into the locales

**Review by stavares843 — CHANGES_REQUESTED** [review]

**Review by stavares843 — APPROVED** [review]

**Review by WanderingHogan — APPROVED** [review]

**Pull request merged**